SPECS/btrfs-progs/btrfs-progs.spec

```diff
@@ -1,16 +1,18 @@
 Name:       btrfs-progs
-Version:    4.19
-Release:        3%{?dist}
+Version:    5.16
+Release:    1%{?dist}
 Summary:    Userspace programs for btrfs
 Group:      System Environment/Base
 License:    GPLv2+
 URL:        http://btrfs.wiki.kernel.org/index.php/Main_Page
-Source0:    https://www.kernel.org/pub/linux/kernel/people/kdave/btrfs-progs/%{name}-v%{version}.tar.xz
+Source0:    https://www.kernel.org/pub/linux/kernel/people/kdave/btrfs-progs/%{name}-v%{version}.tar.xz#/%{name}-%{version}.tar.xz
 %define sha1 btrfs-progs=df4d34b8ecf5eaac177a0b121b41e67fce9612e1
```

Comment:
Nit: these sha1 lines are not needed in the context of our build system, and can be removed